src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -29,9 +29,6 @@ internal interface ISyntaxFactsService : ILanguageService
 
         SyntaxToken ParseToken(string text);
 
-        bool IsAwaitKeyword(SyntaxToken token);
```

Comment:
I am presuming these all just move to SyntaxKinds service?

Reply:
yup